packages/cli/src/commands/validator/signed-blocks.ts

```diff
@@ -60,13 +60,17 @@ export default class ValidatorSignedBlocks extends BaseCommand {
   ]
 
   async run() {
+    const isCel2 = await this.isCel2()
+
+    if (isCel2) {
+      this.error('This command is not available on L2 and might be removed in the future')
```

Comment:
I have some thoughts on this that are not quite formed but something like this.

'on L2' for an external user error I think isnt the most correct wording. maybe its more like "after the CEL2 hardfork"

and if its not available now why would it not already be removed? (thinking from users perspective)

so maybe drop that or we could say "This command is not currently available but might be in the future when decentralized sequencing is enabled"